### PR TITLE
Add Dockerfile for ffmpeg instrumentation

### PR DESCRIPTION
**Pull request by surovic**

### CODE REVIEW

File: examples/Dockerfile-ffmpeg.demo [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Create a separate image with the latest source
+FROM ubuntu:focal AS ffmpeg-sources
+WORKDIR /polytracker/the_klondike
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install -y git && git clone https://git.ffmpeg.org/ffmpeg.git
+
+# Now, build the openjpg image using previously downloaded source
```
Comment (hbrodin) [inline examples/Dockerfile-ffmpeg.demo:7]: ffmpeg

File: examples/Dockerfile-ffmpeg.demo [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+FROM trailofbits/polytracker:latest
+LABEL org.opencontainers.image.authors="marek.surovic@trailofbits.com"
```
Comment (hbrodin) [inline examples/Dockerfile-ffmpeg.demo:9]: Nice that you've adopted Dockers new scheme for authors/maintainers!

**Review by hbrodin — APPROVED** [review]
Ideally, this PR would be split into two:
1. Dockerfile
2. Refactored src
but hey, LGTM. 😄

**Comment by surovic** [comment]
> Ideally, this PR would be split into two:
> 
> 1. Dockerfile
> 2. Refactored src
>    but hey, LGTM. smile

Ahh. I actually have the dedupe separated in a branch, but I merged it into this. Oh well.

**Pull request merged**